src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/InterestForecastingRequests.scala

```diff
@@ -23,12 +23,42 @@ import uk.gov.hmrc.performance.conf.ServicesConfiguration
 
 object InterestForecastingRequests extends ServicesConfiguration {
 
+//  Assumptions
```

Comment:
Fine for now... Ongoing, assumptions will be new for each story. If tests for each story are to be added here then it might makes sense to add them for each tests maybe?

Reply:
maybe we can take out the assumptions and have these in acceptance but always remember to performance tests when these changes?

Reply:
Sounds good. The tests will evolve as more get added